src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -447,6 +447,13 @@
     "NAME": "Scorching Ray",
     "DESCRIPTION": "Remember stsjorbsmod:Lust. NL Apply !stsjorbsmod:MetaMagic! stsjorbsmod:Burning !M! times. This stsjorbsmod:Burning is affected by Strength."
   },
+  "stsjorbsmod:Seance": {
+    "NAME": "Seance",
+    "DESCRIPTION": "Put a card from your exhaust pile into your hand. It gains Ethereal. NL Exhaust",
```

Comment:
`Ethereal` should be `stsjorbsmod:Ethereal` (here and in extended desc)

Reply:
I believe you're thinking of Ephemeral. According to the spec sheet, this should actually become Ethereal, but maybe Ephemeral makes more sense?

Reply:
Oh, I'm sorry, you're right; I was thinking of Ephemeral